htdocs/install/upgrade2.php

```diff
@@ -4222,7 +4222,7 @@ function migrate_delete_old_dir($db, $langs, $conf)
  * @param	Conf		$conf			Object conf
  * @param	array		$listofmodule	List of modules, like array('MODULE_KEY_NAME'=>', $reloadmode)
  * @param   int         $force          1=Reload module even if not already loaded
- * @return	int							<0 if KO, >0 if OK
+ * @return	int|void					<0 if KO, >0 if OK
```

Comment:
The problem that trigger the codding error here is that the first return of the function is empty (return;) line 4230.
Maybe replace return; by return 0; could be a better fix so if the function does something then >0 for success , <0 for error and if nothing done then 0